api/apps/api/src/modules/scenarios/dto/start-scenario-blm-calibration.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsOptional } from 'class-validator';
+import { IsValidRange } from '@marxan-api/decorators/is-valid-range.decorator';
+
+export class StartScenarioBlmCalibrationDto {
+  @ApiProperty({ example: [0.001, 100], required: false })
+  @IsOptional()
+  @IsArray()
+  @IsValidRange()
+  range!: [number, number];
```

Comment:
if `IsOptional`, then `range?: [number, number]`, otherwise we are cheating TS engine vs real incoming requests.

Reply:
Fixed!